.github/ISSUE_TEMPLATE/data_request.md

```diff
@@ -0,0 +1,38 @@
+---
+name: Data Request
+about: Suggest a data set for ingestion in the platform
+title: ''
+labels: 'MSFC'
+assignees: ''
+
+---
+
+**Dataset Name**
+What is the name of the data requested.
+
+**Dataset Description**
+Short description of the data.
```

Comment:
Here and elsewhere in this file I think either this should be a directive:

> Please provide a short description of the data.

or the prompt should not end in a period.

Reply:
Plus one to Aimee's comment.

Reply:
Being Southern, I generally prefer to use the word 'Please' :)

Reply:
Is it a prompt or a replace this text with actual value?